cvat_reader/_base.py

```diff
@@ -179,18 +179,20 @@ def __init__(
                     self.tracks.append(
                         Track(track_id=track_id, annotations=annotations)
                     )
+        self.video_file = video_file
 
         if load_video:
             from .video_reader.cv2_reader import CV2Reader
 
-            self.video_reader: VideoReader = CV2Reader(video_file)
+            self.video_reader: CV2Reader = CV2Reader(video_file)
+            self.last_frame_id = int(self.video_reader.capture.get(7))
```

Comment:
Does cv2 provide a constant for the 7 value?

Reply:
Do you mean something like `cv2.CV_GET_PROP_FRAME_COUNT`?

Reply:
Jup. Easier to reason about what’s going on.